src/include/openenclave/internal/backtrace.h

```diff
@@ -13,6 +13,26 @@ OE_EXTERNC_BEGIN
 /* Maximum backtrace size */
 #define OE_BACKTRACE_MAX 32
 
+/**
+ * This function is intended to be called by GNU **backtrace** (oelibc) and
+ * **oe_backtrace** (oecore) functions.
+ */
+int oe_backtrace_impl(void** start_frame, void** buffer, int size);
```

Comment:
Does this mean that backtraces will be working inside the enclave?

Reply:
These functions need to read ELF headers to be useful.  If we're actually pulling them in, then that means that we're also getting the OE libunwind, which I'd really rather we didn't have (at least, not in release builds) because it's made entirely out of useful gadgets for attackers.

Reply:
No. All that this means is, that I copied over an internal OE header file without looking at it. It's possible that it works, but I never tried it or thought about it.

Reply:
@davidchisnall Yeah, agreed. I don't think I used this particular header for any of my needs, but I think in general we should try to get rid of as many dependencies on internal OE headers as possible.